src/components/nfts/NFTCardGallery.scss

```
@@ -0,0 +1,91 @@
+#card-gallery-container {
+  width: 280px;
+  padding: 15px 15px 24px 15px;
+  background-color: #21383d;
```

In the file [index.scss](https://github.com/CityOfZion/dora/blob/6167155c400e66960685c636bdb39ec37ae02ec5/src/index.css#L52) has the defined colors;

```suggestion
  background-color: var(--darkest-grey);
```